### PR TITLE
Added a new package idds-common for LSST/PanDA integration project

### PR DESCRIPTION
**Pull request by yesw2000**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by yesw2000** [comment]
@beckermr,
Please help take a look. All the platform checks failed with dependency error. Even the local run of "**conda-build**" for this package idds-common succeeded.

Shuwei

File: recipes/idds-common/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - flake8
+    - nose
+    - pep8
+    - pip
+    - pytest
```
Comment (beckermr) [inline recipes/idds-common/meta.yaml:27]: These all look like test deps and pip is not usually needed at run time. Try removing those and unittest2.

**Comment by beckermr** [comment]
nose pep8 and flake8 are test deps too

File: recipes/idds-common/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+  run:
+    - flake8
+    - nose
+    - pep8
+    - pytest
```
Comment (beckermr) [inline recipes/idds-common/meta.yaml:25]:
```suggestion
```

File: recipes/idds-common/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: d9d205ca74453a0398491e096f885ae50294f326818d58ef9bc101e6e8db8aca
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
```
Comment (beckermr) [inline recipes/idds-common/meta.yaml:20]:
```suggestion
    - python >=3.6
    - pip
```

**Comment by beckermr** [comment]
win failure is fine

**Comment by yesw2000** [comment]
> 
> 
> win failure is fine

It is possible to ask github to disable win platform check?

And who is in charge of approving the pull request?

**Comment by beckermr** [comment]
I can approve and merge. No need to disable the check. This PR will generate a new feedstock and add you. In the feedstock the PR will only build on linux-64. Please see our documentation for more details. https://conda-forge.org/docs/maintainer/adding_pkgs.html

**Comment by yesw2000** [comment]
@beckermr 
Why is the package **idds-common** still **unavailable** to conda?

> % conda search -c conda-forge idds-common
Loading channels: done
No match found for: idds-common. Search: *idds-common*
> 
> PackagesNotFoundError: The following packages are not available from current channels:
>
> - idds-common
>
> Current channels:
>
> - https://conda.anaconda.org/conda-forge/linux-64
>  - https://conda.anaconda.org/conda-forge/noarch
>  - https://repo.anaconda.com/pkgs/main/linux-64
>  - https://repo.anaconda.com/pkgs/main/noarch
>  - https://repo.anaconda.com/pkgs/r/linux-64
>  - https://repo.anaconda.com/pkgs/r/noarch

Did I miss anything?

**Comment by beckermr** [comment]
It takes a a few hours for things to go live. Plus we may have a cdn outage blocking things.

**Pull request merged**